RegExp.make.js

```diff
@@ -971,11 +971,12 @@ RegExp.make = (function () {
   }
 
   function make(flags, template, ...values) {
+    // make is always bound to a flags string
     if ('string' === typeof template && values.length === 0) {
       // Allow RegExp.make(i)`...` to specify flags.
       // This calling convention is disjoint with use as a template tag
       // since the typeof a template record is 'object'.
-      return make.bind(null, template /* use as flags instead */);
+      return make.bind(null, flags + template /* use as flags instead */);
```

Comment:
won't this cause a problem if i do `make('g')('g')`? or should we expect that to throw, much later, when it's invoked as a tagged template string?